packaging/test_rule_engine_plugin_metadata_guard.py

```diff
@@ -258,7 +260,7 @@ def test_plugin_cannot_be_bypassed_with_addw(self):
             with lib.file_backed_up(config.server_config_path):
                 self.enable_rule_engine_plugin(config)
 
-                self.user.assert_icommand(['imeta', 'addw', '-C', '%', 'irods::addw', 'v'], 'STDERR', ['CAT_INSUFFICIENT_PRIVILEGE_LEVEL'])
+                self.user.assert_icommand(['imeta', 'addw', '-C', self.user.session_collection, 'irods::addw', 'v'], 'STDERR', ['CAT_INSUFFICIENT_PRIVILEGE_LEVEL'])
```

Comment:
is this still the correct error - `CAT_INSUFFICIENT_PRIVILEGE_LEVEL`?

Reply:
I believe so: the tests pass, and it's what I got manually as well.

Reply:
okay - fair enough.  thanks.   resolving.